src/Network/Receive.pm

```diff
@@ -3705,7 +3705,7 @@ sub shop_sold_long {
 sub vending_start {
 	my ($self, $args) = @_;
 
-	my $item_pack = $self->{vender_items_list_item_pack_self} || $self->{vender_items_list_item_pack} || 'V v2 C v C3 a8';
+	my $item_pack = $self->{vender_items_list_item_pack} || 'V v2 C v C3 a8';
```

Comment:
this will cause parser errors and close openkore, in some servers like bRO, unpack string to self vending items is different compared to other vending items.
and some servers like kRO de unpack string is the same

Reply:
reference:
https://github.com/OpenKore/openkore/pull/2346/files

Reply:
I'm sorry, I haven't found this code for some reason before